php/class-customize-snapshot-manager.php

```diff
@@ -640,6 +645,62 @@ public function render_templates() {
 
 			<# } #>
 		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-button">
+			<?php $title_text = __( 'has potential conflicts (click to expand)', 'customize-snapshots' ); ?>
+			<# id= data.setting_id.replace( /\]/g, '\\]' ).replace( /\[/g, '\\[' ); #>
+			<span>
+				<?php esc_html_e( 'Conflicts', 'customize-snapshots' ); ?>
+				<a href="<?php echo esc_url( '#TB_inline?width=600&height=550&inlineId=snapshot-conflicts-' ); ?>{{id}}" class="dashicons dashicons-warning thickbox snapshot-conflicts-button" title="{{id}} <?php echo esc_attr( $title_text ); ?>"></a>
+			</span>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-notification-template">
+			<ul>
+				<# _.each( data.notifications, function( notification ) { #>
+					<li class="notice notice-{{ notification.type || 'info' }} {{ data.altNotice ? 'notice-alt' : '' }}" data-code="{{ notification.code }}" data-type="{{ notification.type }}">{{{ notification.message || notification.code }}}</li>
+				<# } ); #>
+			</ul>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict">
+			<div id="snapshot-conflicts-{{data.setting_id}}" class="snapshot-conflict-thickbox-content thickbox">
+					<# _.each( data.conflicts, function( setting ) { #>
+						<details class="snapshot-conflict-details">
+							<summary>
+								<code>{{setting.uuid}}
+									<# if ( ! _.isEmpty( setting.name ) ) {
+										if ( ! _.isEmpty( setting.uuid ) ){ #>
+											-
+										<# } #>
+										{{setting.name}}
+									<# } #>
+								</code>
+								<# if ( ! _.isEmpty( setting.edit_link ) ) { #>
+									<a target="_blank" href="{{setting.edit_link}}" class="dashicons dashicons-external"></a>
+								<# } #>
+							</summary>
+							<article class="snapshot-value">
+								{{{setting.value}}}
+							</article>
+						</details>
+					<# }); #>
+			</div>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-conflict-value">
+			<# if ( _.isEmpty( data.value ) ) { #>
+					<em><?php esc_html_e( '(Empty String)', 'customize-snapshots' ); ?></em>
+			<# } else if ( _.isString( data.value ) ||  _.isNumber( data.value ) ) { #>
+					<p>{{data.value}}</p>
+			<# } else if ( _.isBoolean(data.value) ) {
+				var temp = JSON.stringify(data.value); #>
+					<p>{{temp}}</p>
+			<# } else {
+				var temp = JSON.stringify( data.value, null, 4 ); #>
```

Comment:
Why the `temp` variable here and above? You should be able to do just `{{ JSON.stringify( data.value, null, 4 ) }}`